components/python/automat/test/results-3.5-32.master

```diff
@@ -0,0 +1,5 @@
+_test/test_core.py .....                                                 [  8%]
+_test/test_discover.py .......................                           [ 45%]
+_test/test_methodical.py .................                               [ 73%]
```

Comment:
running the tests here gives
-_test/test_discover.py .......................                           [ 45%]
+_test/test_discover.py sssssssssssssssssssssss                           [ 45%]

Reply:
Who knows which other python modules these tests use... :S